### ISSUE TITLE
Failing with `Text copy failed` message for `Copy text from image`

### ISSUE THREAD

**Issue by MadhaviSeelam** [brave/brave-browser#30203]
## Steps to Reproduce

- Install 1.52.91
- launch Brave
- navigate to search.brave.com in a new tab
- search for `blue jay bird` 
- right click on the image to open context menu

## Actual result:

Copying for text message` message is shown for a brief second before showing `Text copy failed` dialog is shown.
ex|ex|ex
---|---|---
![image](https://user-images.githubusercontent.com/98358127/236878172-d80b6b25-142c-436c-8a43-d28688530605.png)| ![image](https://user-images.githubusercontent.com/98358127/236878364-a3cc2194-bedc-469e-81b5-cb257369c34e.png)| ![image](https://user-images.githubusercontent.com/98358127/236878305-b74c38f4-07ba-4606-9118-520fa190c942.png)

## Expected result:
Extracted text should be copied to clipboard
Same expected results should show for thumbnail as well.

## Reproduces how often: 

Easily

## Brave version (brave://version info)

```
Brave | 1.52.91 Chromium: 113.0.5672.77 (Official Build) beta (64-bit)
-- | --
Revision | c4236862955e005c2187105415ac4a2ecf86dff1-refs/branch-heads/5672_62@{#3}
OS | Windows 11 Version 22H2 (Build 22621.1555)
```

## Version/Channel Information:

- Can you reproduce this issue with the current release? N/A
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:

@simonhong @rebron 
cc: @brave/qa-team

**Comment by rebron** [comment]
@MadhaviSeelam What's the expected text from the image? 
Looks like just a picture of jay bird with no text. We can maybe improve output text if there's no text to copy. 

"Text copy failed. This image does not contain any text."

**Comment by MadhaviSeelam** [comment]
@rebron you are right there is no text to `blue jay bird` image but improvement would be great as you mentioned.
However, here is an image that has text and text extraction worked fairly well (not 100%), which is fine, I think. I've compared with `OCR - Image Reader` extension in Chrome and Brave's text extraction is better.

Brave | Chrome (OCR extension)
---- | -----
![image](https://github.com/brave/brave-browser/assets/98358127/6255bf52-8a32-4f77-a5e4-f27df3b0b6ab)|![image](https://github.com/brave/brave-browser/assets/98358127/acecd1f3-224d-452e-b5de-fd1e2772afe0)

**Comment by MadhaviSeelam** [comment]
@rebron Here is a simple one.  `Copy text from image` is working as expected in this case. (I don't think translation is working correctly.


https://github.com/brave/brave-browser/assets/98358127/9b5fdf19-84b5-4f22-986e-68492e54d1a9
